New streamer-wands features

teh60 opened the pull request:
- Added missing spells and wands for main branch
- Added more info for spell tooltips
- Added noita beta branch content and apotheosis content
- Added noita item inventory support
- Added noita progress support
- Fixed hover tooltips clipping with window edges

My data scraping is not included in this since they are currently a mess of around 8 python files that all do similar things but aren't built up with well defined functions to make easy user execution.  (If its desired so other people can maintain the project/etc. reasons I can work on getting them into a streamlined form.)

I put some amount of credits in readme.md, but feel free to change/remove if desired.  

My level of testing/compatibility for the site was very limited because I just hosted on my laptop and twitch API was being weird so I only tested on my local network between laptop and desktop.

As always, let me know on here or discord if there's anything else or any issues that I need to resolve.

Commit:
Main commit for teh60's streamer-wand features

- Added missing spells and wands for main branch
- Added more info for spell tooltips
- Added noita beta branch content and apotheosis content
- Added noita item inventory support
- Added noita progress support
- Fixed hover tooltips clipping with window edges

Commit:
changed max number of items to 4, not 16 like inv.

Commit:
fixed apotheosis enemy progress and volcanic egg

Commit:
removed old commented code from public/stats.js

Commit:
fixed two small bugs

- spell progress table tooltips lost their minimum width due to classes
- website was still using dev mode vue, added comment and both so we can easily swap between the two

Commit:
Fixed seed tool URL being wrong

myndzi commented on views/index.pug:
```diff
@@ -21,7 +21,19 @@ block content
+        tracking to start from your current enemy progress follow the below steps:<br><br>
+        1. Browse to your <strong>%appdata%/LocalLow/Nolla_Games_Noita/save00/stats/_stats.salakieli</strong><br>
+        2. Download the stats.lua file<br>
+        3. Move the stats.lua inside streamer-wands, root directory so right next to token.lua and mod.xml<br>
```
I'm going to have to spend a little time on code review, probably will roll it into a stream with the final testing of NT delayed graceful shutdown. I skimmed the changes, and from a user POV this stood out to me as a minor improvement - the instructions here don't explicitly tell the user what to do. It says "browse to", but not in relation to "select the file using the input below". Probably something like this would be a simple but meaningful help:

```suggestion
        1. Click "browse" on the file selector below
        1. Browse to your <strong>%appdata%/LocalLow/Nolla_Games_Noita/save00/stats/_stats.salakieli</strong><br>
        1. Download the stats.lua file<br>
        1. Move the stats.lua inside streamer-wands, root directory so right next to token.lua and mod.xml<br>
```

(The format shift to `1.` thing for everything is just a markdown thing, and intentional - `1.` formats an ordered list, but the actual numerical value doesn't matter. Makes it easier to edit without having to relabel all the numbers)

teh60 committed:
Update views/index.pug

add clearer instructions for streamer stats uploading

Co-authored-by: Kris Reeves <github.com@masquerader.com>

Commit:
fixed icon descriptions and links

Commit:
Merge branch 'main' of https://github.com/teh60/streamer-wands-backend

myndzi committed:
Log path of request errors

myndzi committed:
Use express.static when NO_NGINX env var is "true"

Commit:
created blank stats.lua

myndzi committed:
Improve logging, auto-formatting

myndzi committed:
Infer the websocket scheme (ws/wss) from document.location (http/https)

myndzi committed:
Update items information from websocket messages

myndzi committed:
Remove sticky header/footer, clarify stats instructions, update styles

(Remove the body font, minor style tweaks)

myndzi committed:
Don't update last-sent json unless we've successfully sent data

The mod, when reconnecting, would not update the backend if it had
previously run the function to send an update, even if that function
did nothing because the socket was disconnected. Upon reconnect, no
update would be sent since the code thought it had already done so.

myndzi committed:
Layout and style tweaks (collab with teh60 in stream)

myndzi committed:
Reduce cache-busting copy-paste

myndzi committed:
Bump version

teh60 committed:
Merge pull request #1 from myndzi/myndzi/prep-update

Functional fixes and style tweaks

Commit:
fixed version data not being updated by websocket

Commit:
added cache busting to main.js
